Automattic/jetpack
# WordPress 5.8 Compatibility: Add support for `block_editor_settings_all`

The filter `block_editor_settings` is [being deprecated in favor of `block_editor_settings_all`](https://core.trac.wordpress.org/ticket/52920) for WordPress 5.8.

#### Changes proposed in this Pull Request:

* Adds support for block_editor_settings_all when running WordPress 5.8 or higher. 

#### Jetpack product discussion


https://github.com/Automattic/jetpack/issues/19654

#### Does this pull request change what data or activity we track or use?


No.
#### Testing instructions:






* Smoke test using the block editor.
* Enable the option allowing you to copy posts under Jetpack -> Writing -> Compose settings.
* Make sure the copy post option still works.
